tests/src/test/scala/akka/kafka/scaladsl/TransactionsSpec.scala

```diff
@@ -280,9 +381,13 @@ class TransactionsSpec extends SpecBase(kafkaPort = KafkaPorts.TransactionsSpec)
 
   private def valuesProbeConsumer(settings: ConsumerSettings[String, String],
                                   topic: String): TestSubscriber.Probe[String] =
+    valuesSource(settings, topic)
+      .runWith(TestSink.probe)
+
+  private def valuesSource(settings: ConsumerSettings[String, String],
```

Comment:
Does the probe consumer properties enable `isolation.level` `read_committed`?  This will ensure that only successfully committed messages are consumed.

Reply:
Yes, the consumer has that in its settings:

https://github.com/akka/alpakka-kafka/blob/a93b8b70848395790e76920ab112526cd4156b16/tests/src/test/scala/akka/kafka/scaladsl/TransactionsSpec.scala#L380